<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupForm</class>
 <widget class="QWidget" name="SignupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>280</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
    color: black;
    border: 2px solid black;
    padding: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none; 
	 font-weight: bold;
    color: blue;  
    text-decoration: none; 
}

QPushButton:hover {
    color: blue;
    text-decoration: underline;
}
</string>
   </property>
   <property name="text">
    <string>&lt;-- Back to main</string>
   </property>
  </widget>
  <widget class="QLabel" name="signUpLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 15px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Sign-up Form</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 10px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-size: 8px;
	background-color: #7cb7be;
	color: black;
	border: 0px;
	border-radius: 2px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 10px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="typelbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 10px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>User type</string>
   </property>
  </widget>
  <widget class="QLabel" name="emaillbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 10px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="brandlbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 10px;
	font-weight: bold;
	color: #0b5572;
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string>Brand Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #82bfc7;
	font-size: 15px;
	font-weight: bold;
	border-radius: 5px;
	border: 0px;
}
QPushButton:hover {
	background-color: #0b5572;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-size: 8px;
	background-color: #7cb7be;
	color: black;
	border: 0px;
	border-radius: 2px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-size: 8px;
	background-color: #7cb7be;
	color: black;
	border: 0px;
	border-radius: 2px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="userTypeField">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	font-size: 8px;
	background-color: #7cb7be;
	color: black;
	border: 0px;
	border-radius: 2px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandNameField">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-size: 8px;
	background-color: #7cb7be;
	color: black;
	border: 0px;
	border-radius: 2px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #82bfc7;
	font-weight: bold;
	border-radius: 5px;
	border: 0px;
}
QPushButton:hover {
	background-color: #0b5572;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Choose Logo</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoPreviewLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
